ui-v2/app/templates/dc/acls/tokens/index.hbs

```diff
@@ -31,20 +39,28 @@
   <BlockSlot @name="actions">
       <a data-test-create href="{{href-to 'dc.acls.tokens.create'}}" class="type-create">Create</a>
   </BlockSlot>
-  <BlockSlot @name="content">
-{{#if (gt items.length 0) }}
+  <BlockSlot @name="toolbar">
+  {{#if (gt items.length 0)}}
     <SearchBar
+      data-test-intention-filter="true"
       @value={{search}}
       @onsearch={{action (mut search) value="target.value"}}
+      @secondary="sort"
+      @selected={{sort.selected}}
+      @options={{sort.items}}
+      @onchange={{action (mut sortBy) value='target.value'}}
```

Comment:
I updated this to `value='target.selected.value`, but it caused tests to fail.

Reply:
It needs to be `target.selected.key` not `target.selected.value`. For double checking these just use the original `services/index.hbs` as a reference, all of these should be pretty much the same as that one.